GitUI/BranchTreePanel/RepoObjectsTree.cs

```diff
@@ -382,6 +403,18 @@ private void CreateSubmodules()
             _submoduleTree = new SubmoduleTree(rootNode, UICommandsSource);
         }
 
+        private void CreateStashes()
+        {
+            TreeNode rootNode = new(TranslatedStrings.Stashes)
+            {
+                Name = TranslatedStrings.Stashes,
+                ImageKey = nameof(Images.Stash),
+                SelectedImageKey = nameof(Images.Stash)
+            };
+
+            _stashTree = new StashTree(rootNode, UICommandsSource, _refsSource);
```

Comment:
[nit]
```suggestion
            _stashTree = new(rootNode, UICommandsSource, _refsSource);
```

Reply:
When we talked about this feature, we have previously said that the type should be stated if not on left side of '='
These rules should be documented...
I prefer the suggestion, but do not change right now.

Reply:
We shouldn't use `var` if the type is not obvious from the expression. When newing up the type could be omitted.

This is a non-blocking suggestion.